pbin.sh

```diff
@@ -188,4 +213,20 @@ else
 	paste=$(cat)
 fi
 
-printf "%s" "$paste" | pastebin
+
+if [ -z $verbose ];then
```

Comment:
unclear behavior, why with verbose result is copied to xclip/xsel? what's the purpose?